web-app/src/components/header/employer-nav-links.tsx

```diff
@@ -17,7 +17,7 @@ function EmployerNavLinks(props: EmployerNavLinksProps) {
 	if (props.user.company) {
 		companyURL = urlGenerators.vizeCompanyProfileUrl(props.user.company.id);
 	} else {
-		companyURL = "/company/create";
+		companyURL = `/${urlGenerators.queryRoutes.createCompanyProfile}`;
```

Comment:
I changed the path here to use the english equivalent of `/create-company-profile`. Is there any significant difference between `/company/create` or is there any reason we should keep on using `/company/create`?

Reply:
It just seemed more "organized" to me at the time. It does not matter that much.

An example of what I was thinking:
- `/companies` would be the company search page.
- `/companies/45` would be a company profile page. (45 is the company id.)
- `/companies/create` or `/companies/add` would be the create company form.
- `/companies/45/edit` would be a form to edit a company.